File: IOModels/ShopDetailRequest.cs
namespace SmartAlerts.API.IOModels
{
    public class ShopDetailRequest 
    {

        public long UserID { get; set; }

        public string ShopType { get; set; }

        public TimeSpan OpeningTime { get; set; }

        public TimeSpan ClosingTime { get; set; }

        public string? WebsiteURL { get; set; }

        public string ShopManagerName { get; set; }

        public string ShopContact { get; set; } 
        public string? ShopProfilePicURL { get; set; }
        public string Latitude { get; set; } 

        public string Longitude { get; set; }
    }
}

File: main/java/com/smart/alerts/models/ShopDetailRequest.java
package com.smart.alerts.models;

import java.time.LocalTime;

public class ShopDetailRequest {

    public long UserID;
    public String ShopType;
    public String OpeningTime;
    public String ClosingTime;
    public String WebsiteURL;
    public String ShopManagerName;
    public String ShopContact;
    public String ShopProfilePicURL;
    public String Latitude;
    public String Longitude;

    public ShopDetailRequest() {
        UserID = 0;
        ShopType = "";
        OpeningTime = null;
        ClosingTime = null;
        WebsiteURL = null;
        ShopManagerName = "";
        ShopContact = "";
        ShopProfilePicURL = null;
        Latitude = null;
        Longitude= null;
    }
}
